schemas/hand.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "id": "https://immersive-web/gamepad-mapping/0.1.0/hand.schema.json",
+    "type": "object",
+    "properties": {
+        "asset" : {
```

Comment:
Note related to chat: If you stick with a single asset identifier here then I advocate that it *must* be a standardized format.

Reply:
And that's exactly why i was asking you that question in chat :)